$NetBSD: patch-build__tools_make__xml__documentation,v 1.1 2023/12/15 14:18:16 gdt Exp $

Fix non-POSIX \b in sed to POSIX character class.

Not yet reported upstream.

--- build_tools/make_xml_documentation.orig	2023-12-15 13:33:05.565768402 +0000
+++ build_tools/make_xml_documentation
@@ -214,7 +214,7 @@ for subdir in ${mod_subdirs} ; do
 					${XMLSTARLET} val -e -d "${source_tree}/doc/appdocsxml.dtd" "${i}" || { echo "" ; exit 1 ; }
 			fi
 		fi
-		${SED} -r "/^\s*(<[?]xml|<.DOCTYPE|<.?docs)/d" "${i}" >> "${output_file}"
+		${SED} -r "/^[:space:]*(<[?]xml|<.DOCTYPE|<.?docs)/d" "${i}" >> "${output_file}"
 	done
 done
 echo "</docs>" >> "${output_file}"
